TAF_EasyRest/TestFramework/PageComponents/Client/RestaurantMenuComponents/OrderSummaryDetailsPageComponent.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace TestFramework.PageComponents
+{
+    public class OrderSummaryDetailsPageComponent
+    {
+        private IWebDriver driver;
+        private int index;
+
+        public OrderSummaryDetailsPageComponent(IWebDriver driver, int index)
+        {
+            this.driver = driver;
+            this.index = index;
+        }
+
+        #region Elements
+        private IWebElement _currentElement => driver.FindElement(By.XPath($"(//td[@class='MuiTableCell-root-3326 MuiTableCell-body-3328 MuiTableCell-paddingCheckbox-3332']//parent::tr)[{index}]"));
+        private IWebElement _removeButton => _currentElement.FindElement(By.XPath(".//button[@aria-label='Remove item']"));
+        private IWebElement _quantityField => _currentElement.FindElement(By.XPath(".//input[@type='number']"));
```

Comment:
I doubt you can find these elements. They are inside your parent component. But if you click the remove button(line 23) where is the connection to the parent?

Reply:
After Click on this button OrderConfirmationPageComponent is returned, so this element become vanished